### PR TITLE
Added a new feature

### PR DESCRIPTION
**Pull request by Napassawxn**
This feature is for adding an autocomplete into the search box.

### CODE REVIEW

File: lab-exercise-1/NapassawanFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class NapassawanFeature {
+    public void feature() {
+        // this is a new feature
```
Comment (Napassawxn) [inline lab-exercise-1/NapassawanFeature.java:3]: This content is not well-written. please improve it.

Comment (6588041chalisakengkaewpennapa) [reply lab-exercise-1/NapassawanFeature.java:3]: I think This content is not well-written. please improve it.

File: lab-exercise-1/NapassawanFeature.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    }
+}
```
Comment (6588041chalisakengkaewpennapa) [inline lab-exercise-1/NapassawanFeature.java:5]: I think This content is not well-written. please improve it.

Comment (6588041chalisakengkaewpennapa) [inline lab-exercise-1/NapassawanFeature.java:5]: I think you should fix it

File: lab-exercise-1/NapassawanFeature.java (hunk 1 above)
Comment (nekongum) [inline lab-exercise-1/NapassawanFeature.java:3]: I think you should fix it

Comment (JNapat) [reply lab-exercise-1/NapassawanFeature.java:3]: Go go improve

Comment (rmabow03) [reply lab-exercise-1/NapassawanFeature.java:3]: it not complete the feature, can you add the feature in your code.

**Comment by rmabow03** [comment]
it not complete the feature, can you add the feature in your code and change you name class.

File: lab-exercise-1/NapassawanFeature.java (hunk 1 above)
Comment (JNapat) [inline lab-exercise-1/NapassawanFeature.java:3]: u have to add the function detail na

Comment (jaxmmmm) [reply lab-exercise-1/NapassawanFeature.java:3]: good

File: lab-exercise-1/NapassawanFeature.java
Comment (rmabow03) [inline lab-exercise-1/NapassawanFeature.java:1]: it not complete the feature, can you add the feature in your code.

**Pull request merged**